uploader.json

```diff
@@ -4484,6 +4489,15 @@
                 }
             ]
         },
+        "mysqladmin_vars": {
+            "host": [
+                {
+                    "archive_file_name": "/insights_commands/mysqladmin_variables",
+                    "command": "/bin/mysqladmin variables",
+                    "pattern": []
+                }
+            ]
+        },
```

Comment:
@JoySnow please remove this section of the json as it is not being used.  If you don't mind, could you also add the command to `uploader.v2.json` in your PR?